src/sbm.jl

```diff
@@ -916,26 +919,31 @@ function update_until_recharge(sbm::SBM, config)
             sbm.waterlevel_river[i] * sbm.riverfrac[i],
             sbm.riverfrac[i] * sbm.potential_evaporation[i],
         )
-        ae_openw_l = min(
-            sbm.waterlevel_land[i] * sbm.waterfrac[i],
-            sbm.waterfrac[i] * sbm.potential_evaporation[i],
+        # Determine remaining fraction of available potential_evaporation
+        soilevap_fraction = max(
+            sbm.canopygapfraction[i] - sbm.riverfrac[i] - sbm.glacierfrac[i],
+            0.0,
         )
+        # Determine amount of surface water
+        surface_water = sbm.waterlevel_land[i] * (1.0 - sbm.riverfrac[i])
```

Comment:
Would recommend to use the same approach for `runoff_land` (L. 912) for consistency. Here `waterfrac` is still used.

Reply:
This implementation (not using `waterfrac`) results in a hydrograph that is quite flashy (tested by @JoostBuitink ). Most likely this is caused by a disconnect between the overland flow kinematic wave and soil: water is available in the kinematic wave while the soil (unsaturated zone) has also capacity left (`ustorecapacitiy`), especially in the case when infiltration of surface water is not allowed.

So, while we model the kinematic wave routing for the complete land fraction, and allow for open water evaporation and surface runoff for that fraction (by not using `waterfrac`), most likely the soil is not completely saturated. As a consequence, this results most likely in an "overestimation" of surface runoff (`runoff_land`) and open water evaporation.

Based on this, my recommendation would be to:
1) Revert back to using `waterfrac`, the approach is then consistent between surface runoff and open water evaporation, or
2) Use `waterfrac` unless the soil in the grid cell is almost completely saturated (`ustorecapacitiy` $\approx$ 0.0), then use the land fraction.

Another approach (for the longterm) and similar to 2) could be to use subgrid variability, for example by using a pdf of soil storage capacity or of topographic information to derive a dynamic saturated fraction (e.g. see also this [paper](https://agupubs.onlinelibrary.wiley.com/doi/full/10.1029/2007JD008940)).